### PR TITLE
[nemo-qml-plugin-systemsettings] Simplify DBus argument in call chang…

### PR DESCRIPTION
**Pull request by dcaliste**
…ing GPS flight mode.

Nothing serious but while searching for why my GPS was in flight mode, I notice this interesting construction. I may miss something, but I think it can be simplified a bit.

### CODE REVIEW

File: src/locationsettings.cpp [hunk 1]
```diff
@@ -515,7 +515,7 @@ void LocationSettings::setGpsFlightMode(bool flightMode)
     if (d->m_gpsTechInterface) {
         QDBusReply<void> reply = d->m_gpsTechInterface->blockingCall("SetProperty",
                                                              PoweredPropertyName,
-                                                             QVariant::fromValue<QDBusVariant>(QDBusVariant(QVariant::fromValue<bool>(!flightMode))));
+                                                             QVariant::fromValue<bool>(!flightMode));
```
Comment (pvuorela) [inline src/locationsettings.cpp:518]: Didn't test, but wouldn't this now call SetProperty with bool as parameter type instead of dbus variant bool?

Comment (dcaliste) [reply src/locationsettings.cpp:518]: Good point, I need to test. Maybe that's the reason of such complex construction to pass a boolean value in the end.

Comment (pvuorela) [reply src/locationsettings.cpp:518]: Long time. Could just close the pr maybe?

Comment (dcaliste) [reply src/locationsettings.cpp:518]: Yes, indeed.

**Pull request closed without merging**